packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -141,18 +141,40 @@ export class TriggerManager extends Component {
       const konnectorURI = get(konnector, 'vendor_link')
       const konnectorName = get(konnector, 'name') || get(konnector, 'slug')
 
-      const cipherData = {
-        id: null,
-        type: 1,
-        name: konnectorName,
-        login: {
+      let originalCipher = null
+      if (isUpdate) {
+        const id = accounts.getVaultCipherId(account)
+        const search = {
           username: login,
-          password,
-          uris: konnectorURI ? [{ uri: konnectorURI, match: 0 }] : []
+          uri: konnectorURI,
+          type: CipherType.Login
         }
+        const sort = [view => view.login.password === password, 'revisionDate']
```

Comment:
Nit: this looks a bit strange for a sort, maybe worth a comment to explain what we're doing.